pkg/bff/models/v1/registration.go

```diff
@@ -550,39 +550,175 @@ func (r *RegistrationForm) ValidateContacts() error {
+		if _, verr := mail.ParseAddress(email); verr != nil {
+			err = append(err, &ValidationError{Field: fieldName + ".email", Err: ErrInvalidEmail.Error()})
 		}
 	}
 
 	phone := strings.TrimSpace(contact.Phone)
 	if phone == "" {
-		v = append(v, &ValidationError{Field: fieldName + ".phone", Err: ErrMissingField.Error()})
+		err = append(err, &ValidationError{Field: fieldName + ".phone", Err: ErrMissingField.Error()})
 	}
 
 	// TODO: Ensure this is a valid phone number
 
-	return v
+	if len(err) == 0 {
+		return nil
+	}
+
+	return err
 }
 
 // Validate only the fields in the trixo step.
 func (r *RegistrationForm) ValidateTRIXO() error {
-	// TODO: implement
-	return nil
+	err := make(ValidationErrors, 0)
+
+	if r.Trixo == nil {
+		err = append(err, &ValidationError{
+			Field: FieldTRIXO,
+			Err:   ErrMissingField.Error(),
+		})
+		return err
+	}
+
+	// TODO: Validate country name or ISO-3166-1 code
+	if strings.TrimSpace(r.Trixo.PrimaryNationalJurisdiction) == "" {
+		err = append(err, &ValidationError{
+			Field: FieldTRIXOPrimaryNationalJurisdiction,
+			Err:   ErrMissingField.Error(),
+		})
+	}
+
+	if strings.TrimSpace(r.Trixo.PrimaryRegulator) == "" {
+		err = append(err, &ValidationError{
+			Field: FieldTRIXOPrimaryRegulator,
+			Err:   ErrMissingField.Error(),
+		})
+	}
+
+	finTransfers := strings.ToLower(strings.TrimSpace(r.Trixo.FinancialTransfersPermitted))
+	if finTransfers == "" {
+		err = append(err, &ValidationError{
+			Field: FieldTRIXOFinancialTransfersPermitted,
+			Err:   ErrMissingField.Error(),
+		})
+	} else if finTransfers != "yes" && finTransfers != "no" && finTransfers != "partially" {
+		err = append(err, &ValidationError{
+			Field: FieldTRIXOFinancialTransfersPermitted,
+			Err:   ErrInvalidField.Error(),
+		})
+	}
+
+	for i, juris := range r.Trixo.OtherJurisdictions {
+		// TODO: Validate country name or ISO-3166-1 code
+		if strings.TrimSpace(juris.Country) == "" {
+			err = append(err, &ValidationError{
+				Field: FieldTRIXOOtherJurisdictionsCountry,
+				Err:   ErrMissingField.Error(),
+				Index: i,
+			})
+		}
+
+		if strings.TrimSpace(juris.RegulatorName) == "" {
+			err = append(err, &ValidationError{
+				Field: FieldTRIXOOtherJurisdictionsRegulatorName,
+				Err:   ErrMissingField.Error(),
+				Index: i,
+			})
+		}
+
+		if strings.TrimSpace(juris.LicenseNumber) == "" {
+			err = append(err, &ValidationError{
+				Field: FieldTRIXOOtherJurisdictionsLicenseNumber,
+				Err:   ErrMissingField.Error(),
+				Index: i,
+			})
+		}
+	}
+
+	hasReg := strings.ToLower(strings.TrimSpace(r.Trixo.HasRequiredRegulatoryProgram))
+	if hasReg == "" {
+		err = append(err, &ValidationError{
+			Field: FieldTRIXOHasRequiredRegulatoryProgram,
+			Err:   ErrMissingField.Error(),
+		})
+	} else if hasReg != "yes" && hasReg != "no" {
+		err = append(err, &ValidationError{
+			Field: FieldTRIXOHasRequiredRegulatoryProgram,
+			Err:   ErrInvalidField.Error(),
```

Comment:
Same as above, I'd prefer if the error said `field should be 'yes' or 'no'` - but again, this is a dropdown, so it shouldn't cause any problems.